linux/library/src/flutter_embedder.cc

```diff
@@ -99,8 +101,8 @@ static void GLFWOnFlutterPlatformMessage(const FlutterPlatformMessage *message,
     return;
   }
 
-  Json::FastWriter writer;
-  std::string output = writer.write(response);
+  Json::StreamWriterBuilder wbuilder;
```

Comment:
'wbuilder' (and 'rbuilder' above) will similarly need to be changed to meet the style guide guidance on not abbreviating names.